app/views/notifications/_badgeachievement.html.erb

```diff
@@ -9,10 +9,12 @@
 
     <div class="crayons-card crayons-card--secondary p-4 w-100 grid gap-2">
       <img class="w-100 max-w-50 l:max-w-25 h-auto inline-block mx-auto" src="<%= optimized_image_url(json_data["badge_achievement"]["badge"]["badge_image_url"], width: 250) %>" alt="<%= json_data["badge_achievement"]["badge"]["title"] %>">
-      <p class="color-base-70"><em><%= json_data["badge_achievement"]["rewarding_context_message"].html_safe %></em></p>
+      <p class="color-base-70"><em><%= json_data["badge_achievement"]["rewarding_context_message"].to_s.html_safe %></em></p>
     </div>
 
   <p class="self-stretch"><a href="<%= json_data["user"]["path"] %>" class="crayons-btn w-100 m:w-50">Visit your profile</a></p>
-  <p class="max-w-75 mx-auto">You also get <a href="/credits" class="crayons-link crayons-link--brand fw-bold">5 new credits</a> to use for <a href="/listings">community listings</a> if you have anything you'd like to promote. 🎉</p>
-  <p class="fs-s"><a href="<%= about_listings_url %>" class="crayons-link crayons-link--secondary">More information about listings<a></p>
+  <% unless json_data["badge_achievement"]["badge"]["credits_awarded"].zero? %>
```

Comment:
**suggestion (non-blocking):** In a similar vain to the above comments, but also introducing `dig`. Not sure if we also need a safe navigation operator here:

```suggestion
  <% if json_data.dig("badge_achievement", "badge", "credits_awarded").positive? %>
```